rdtools/analysis_chains.py

```diff
@@ -75,6 +75,14 @@ class TrendAnalysis():
         filter_params defaults to empty dicts for each function in rdtools.filtering,
         in which case those functions use default parameter values,  `ad_hoc_filter`
         defaults to None. See examples for more information.
+    filter_params_aggregated: dict
+        parameters to be passed to rdtools.filtering functions that specifically handle
+        aggregated data (dily filters, etc). Keys are the names of the rdtools.filtering functions.
+        Values are dicts of parameters to be passed to those functions. Also has a special key
+        `ad_hoc_filter`; this filter is a boolean mask joined with the rest of the filters.
+        filter_params_aggregated defaults to empty dicts for each function in rdtools.filtering,
+        in which case those functions use default parameter values,  `ad_hoc_filter`
+        defaults to None. See examples for more information.
```

Comment:
We'll need a way to distinguish which filters are aggregated and which are high-frequency. Options that come to mind are to put aggregated filters in their own module or to adopt a naming convention. (e.g. `aggregated_outlier_filter()`) Anyone have a preference?

Reply:
Capturing an update from a meeting: It's possible that some filters would technically work for both high-frequency and aggregated data. The most general option would be to keep everything in the single filtering module, but specify in the TrendAnalysis docstring which keys are appropriate for which attribute. (We don't need to finalize this for this PR)